htdocs/accountancy/journal/bankjournal.php

```diff
@@ -314,7 +314,9 @@
 			}
 			// Now loop on each link of record in bank (code similar to bankentries_list.php)
 			foreach ($links as $key => $val) {
-				if ($links[$key]['type'] == 'user' && !$is_sc) continue;
+				if ($links[$key]['type'] == 'user' && $is_sc) {
```

Comment:
Note: There is no screen to transfer social contribution in a different step than payment of social contribution like we have for invoice. So for social contribution, it seems only payment is transferred.

Pb is here:
![image](https://github.com/Dolibarr/dolibarr/assets/883887/2bf257ea-1340-4193-a9c0-5901d3001516)

When a social contribution is linked to a user, if user has an aux account, the aux account is used as a different line.

@aspangaro 
If we have a social contribution SC with account code SC, paid to user U with account code AUXU, paid in bank B with account code B what should we get in transfer ?

This ?
```
Account    Aux account      Deb   Cred
----------------------------------------------------
B                                                         X
SC               AUXU                X    
```

Reply:
I confirm. For social contribution, only payment are transfered.

User was added to social contribution for statistic but it should have no influence on the generation of the accounting entry. Whether the user is defined or not, he/she should not proceed with the addition of a line containing a subsidiary accounting account.

@eldy  We'll always have:
`Account    Aux account         Deb   Cred
----------------------------------------------------
B                                                        X
SC                                           X`

For me PR is validated

Example:

Without correction

![image](https://github.com/Dolibarr/dolibarr/assets/2341395/cf477021-c9f5-495c-b280-ed92a7edba0a)

With correction

![image](https://github.com/Dolibarr/dolibarr/assets/2341395/837d7a47-b93b-40fc-aedb-79ce1c5dc1ee)